devTools/cardCreator/appData_example/readme.md

```diff
@@ -0,0 +1,22 @@
+## Importing Custom Cards
+ 
+In what follows the "config directory" path refers to one of these locations, depending on your operating system.
+
+* MacOSX: /Users/yourusername/Library/Application Support/cardboard/1.0
+* Windows: C:\Users\yourusername\AppData\Roaming\projectcardboard\cardboard\1.0
+* Linux: /home/yourusername/.config/cardboard/1.0
+
+## Basic overview
+
+Step (1): Design the card in json format
+Step (2): Add the card to the "Custom_Cards" directory
+Step (3): Open up the command line/powershell and run the .exe
+	cmd chdir <this directory>
+	cmd ./AddJsonCardToGame_appData.exe
+Step (4): If the .exe highlights any errors, try to correct them. Otherwise move directly to step(5)
+Step (5): Add this card to the "Custom_Cards" and "Custom_Cards_Sprites" folder in the config directory (if this is your first custom card, you will need to create these directories).
```

Comment:
the cardName.minion.json file goes into the Custom_Cards folder, the plist.json and .png files go into the Custom_Card_Sprites folder.